codec-http/src/test/java/io/netty/handler/codec/http/HttpUtilTest.java

```diff
@@ -130,12 +134,42 @@ public void testGetMimeType() {
     }
 
     @Test
-    public void testGetContentLengthDefaultValue() {
-        HttpMessage message = new DefaultHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.OK);
-        assertNull(message.headers().get(HttpHeaderNames.CONTENT_LENGTH));
+    public void testGetContentLengthThrowsNumberFormatException() {
+        final HttpMessage message = new DefaultHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.OK);
+        message.headers().set(HttpHeaderNames.CONTENT_LENGTH, "bar");
+        try {
+            HttpUtil.getContentLength(message);
+            fail();
+        } catch (final NumberFormatException e) {
+            // a number format exception is expected here
+            assertThat(e, hasToString(containsString("For input string: \"bar\"")));
```

Comment:
can you please avoid checking for string contents. This may vary based upon local and is generally fragile. IMHO checking for `NumberFormatException` should be "good enough" and not subject to false failures if implicitly defined strings change behind the scenes.